libs/pilight/protocols/433.92/bresser_3ch.c

```diff
@@ -12,27 +18,23 @@
 #include "../../core/gc.h"
 #include "bresser_3ch.h"
 
-
 #define MIN_PULSE_LENGTH    14 //488
 #define MAX_PULSE_LENGTH    57 //1950
-#define AVG_PULSE    42 //(1950+965)/2
+#define AVG_PULSE    		42 //(1950+965)/2
```

Comment:
What's your tabs size 'cause indentation is still off here.

This is how it should be:
```
#define MIN_PULSE_LENGTH    14 //488
#define MAX_PULSE_LENGTH    57 //1950
#define AVG_PULSE    				42 //(1950+965)/2

#define RAW_LENGTH        	74
#define MIN_RAW_LENGTH			130
#define MAX_RAW_LENGTH      255
#define MESSAGE_LENGTH      36
#define SYNC_LENGTH  				3890
 
#define SEP_LENGTH   				488
#define BIT1_LENGTH  				1950
#define BIT0_LENGTH  				965
#define RING_BUFFER_SIZE  	256
```

Reply:
sorry, i missed the whitespaces.